pkg/config/config.go

```diff
@@ -997,10 +997,16 @@ func (h *hookConfig) validate(hookType string) (err error) {
 	} else if parsed, err := url.Parse(h.URI); err != nil {
 		return errors.Errorf("failed to parse template url: %w", err)
 	} else if !(parsed.Scheme == "http" || parsed.Scheme == "https" || parsed.Scheme == "pg-functions") {
-		return errors.Errorf("Invalid HTTP hook config: auth.hook.%v should be a Postgres function URI, or a HTTP or HTTPS URL", hookType)
+		return errors.Errorf("Invalid auth hook config: auth.hook.%v should be a Postgres function URI, or a HTTP or HTTPS URL", hookType)
+	} else if (parsed.Scheme == "http" || parsed.Scheme == "https") && h.Secrets == nil {
+		return errors.Errorf("Invalid HTTP config: auth.hook.%v missing required secret value for the http hook endpoint.", hookType)
```

Comment:
**remark**

Splitted up the URI and Secret checking in two different error message since secrets is only required for http endpoints.